app/src/common/shared/org/mozilla/vrbrowser/browser/SessionStore.java

```diff
@@ -906,6 +904,30 @@ public void setMultiprocess(final boolean enabled) {
         }
     }
 
+
+    private State getCurrentState() {
+        if (mCurrentSession != null) {
+            return mSessions.get(mCurrentSession.hashCode());
+        }
+        return null;
+    }
+
+    public void setMultiprocess(final boolean aEnabled) {
+        State state = getCurrentState();
+        if (state != null && state.mSettings.multiprocess != aEnabled) {
+            state.mSettings.multiprocess = aEnabled;
+            recreateSession(state.mSettings);
+        }
+    }
+
+    public void setTrackingEnabled(final boolean aEnabled) {
+        State state = getCurrentState();
+        if (state != null && state.mSettings.trackingProtection != aEnabled) {
+            state.mSettings.multiprocess = aEnabled;
```

Comment:
This seems to be the problem, looks like a CCP error. `multiprocess` should be `trackingProtection`